### DISCUSSION TITLE
--dualdevices 1,2,3 not applicable

### DISCUSSION THREAD

**Discussion by Singam322** [Lolliedieb/lolMiner-releases#1073] [Q&A]
Hi,
Looks like --dualdevices 1,2,3 not applicable. Trying use it alone and with --devices, but no effect.
I have 4 GPUs and need to block TON for GPU 0, but there is no response to the command.
At the same time, the --devices works fine separately.
Can you explain how it work or maybe it's a bug?

**Comment by eaamgh** [comment]
According to the guides, if I'm not mistaken, --dualdevices does not apply when using DUALTON mode. I did what you want by using different miners. One  miner on GPU's 1,2,3 and lolminer on GPU 0 in DUALTON mode.

**Reply by Masakatra** [reply]
how do you do that?

**Reply by eaamgh** [reply]
No longer an issue. lolMiner 1.44 now allows --dualdevices with TONDUAL mode

**Reply by Singam322** [reply]
Absolutely right. For my 4 GPUs rig I needed to stop ton mining on GPU 0 (RX 580 8GB), so --maxdualimpact 0,*,*,* helps me to solve this. It seems strange but it works.

**Reply by cypher619** [reply]
to stop a GPU from Dual Mining use the --maxdualimpact parameter.  

--maxdualimpact * , * , * ,0
_note there are no spaces in the usage.  I used spaces here because the editor deleted the * when right next to a comma_

-    0 = don't mine second coin
-  *= use default mining parameter.

You can also use numbers to impact dual mining performance on second coin.  See the wiki for more information on that.
